habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -291,3 +289,76 @@ def append_text_to_image(image: np.ndarray, text: str):
     text_image = blank_image[0 : y + 10, 0:w]
     final = np.concatenate((image, text_image), axis=0)
     return final
+
+
+def append_text_to_image(
```

Comment:
What is the difference between `append_text_to_image` and `append_text_underneath_image` ?  Can these be the same method with an extra argument? Why do we need both?

Reply:
I didn't wont to change the way function were impemented in scope of this PR. But, you are right that they do very similar things (one overlays text on image and the other concatenates text as image with black background to observation image).

Reply:
Let's rename the append_text_to_image to overlay_text_to_image then ?
If they do different things, we should keep both, but we need to document them properly and give them good names.

Reply:
Done